layouts/css/page-modules/_header.scss

```diff
@@ -155,6 +167,11 @@ header {
         margin: 0;
       }
     }
+    .logo{
```

Comment:
I wonder if 481px is the "right" size for min-width. I'm pretty sure on a screen with 1080px of width it will also look weird

Reply:
Idk I use the old break point